http/headers/Set-Cookie.json

```
@@ -142,10 +142,13 @@
               "oculus": "mirror",
               "opera": "mirror",
               "opera_android": "mirror",
-              "safari": {
-                "version_added": "18.4",
-                "version_removed": "18.5"
-              },
+              "safari": [
+                {
+                  "version_added": "18.4",
+                  "version_removed": "18.5",
+                  "impl_url": "https://webkit.org/b/292975"
+                }
+              ],
```

For single statements, we just use the object, not an array.

(Note: If you run `npm run lint:fix` locally, it should automatically fix this for you.)

```suggestion
              "safari": {
                "version_added": "18.4",
                "version_removed": "18.5",
                "impl_url": "https://webkit.org/b/292975"
              },
```